src/rp2040/busctrl/regs.rs

```diff
@@ -77,18 +106,38 @@ impl Default for BusPriorityAck {
         BusPriorityAck(0)
     }
 }
-#[doc = "Bus fabric performance counter 2"]
+impl core::fmt::Debug for BusPriorityAck {
+    fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result {
+        f.debug_struct("BusPriorityAck")
+            .field("bus_priority_ack", &self.bus_priority_ack())
+            .finish()
+    }
+}
+#[cfg(feature = "defmt")]
+impl defmt::Format for BusPriorityAck {
+    fn format(&self, f: defmt::Formatter) {
+        #[derive(defmt :: Format)]
+        struct BusPriorityAck {
+            bus_priority_ack: bool,
+        }
+        let proxy = BusPriorityAck {
+            bus_priority_ack: self.bus_priority_ack(),
+        };
+        defmt::write!(f, "{}", proxy)
+    }
+}
+#[doc = "Bus fabric performance counter 0"]
 #[repr(transparent)]
 #[derive(Copy, Clone, Eq, PartialEq)]
 pub struct Perfctr(pub u32);
 impl Perfctr {
-    #[doc = "Busfabric saturating performance counter 2 Count some event signal from the busfabric arbiters. Write any value to clear. Select an event to count using PERFSEL2"]
+    #[doc = "Busfabric saturating performance counter 0 Count some event signal from the busfabric arbiters. Write any value to clear. Select an event to count using PERFSEL0"]
```

Comment:
For example: Here 2 was replaced with 0.

Reply:
yep, this is expected, it's ok